docs/docs/deploying.md

```diff
@@ -6,27 +6,28 @@ Deploying Quirrel is straight-forward.
-| `QUIRREL_TOKEN`             | access token for the Quirrel server.                                    | **Hosted:** Create a new Project + Client in the [Dashboard](https://quirrel.dev/dashboard). **Self-Hosted:** Check [How to deploy our own server](#how-to-deploy-your-own-server) .         |
-| `QUIRREL_BASE_URL`          | The base URL of your application's deployment.                               | You probably know this. Something like `my-application.com`.                                                                                                                     |
-| `QUIRREL_ENCRYPTION_SECRET` | A 32-character-long secret used for end-to-end encryption of your jobs. | Can be generated using `openssl rand -hex 16` or [random.org](https://www.random.org/strings/?num=2&len=16&digits=on&upperalpha=on&loweralpha=on&unique=on&format=html&rnd=new). |
+| Variable                    | Meaning                                                                 | Where to get                                                                                                                                                                         |
+| --------------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `QUIRREL_TOKEN`             | access token for the Quirrel server.                                    | **Hosted:** Create a new Project + Client in the [Dashboard](https://quirrel.dev/dashboard). **Self-Hosted:** Check [How to deploy our own server](#how-to-deploy-your-own-server) . |
+| `QUIRREL_BASE_URL`          | The base URL of your application's deployment.                          | You probably know this. Something like `my-application.com`.                                                                                                                         |
+| `QUIRREL_ENCRYPTION_SECRET` | A 32-character-long secret used for end-to-end encryption of your jobs. | Can be generated using `openssl rand -hex 16` or [random.org](https://www.random.org/strings/?num=2&len=16&digits=on&upperalpha=on&loweralpha=on&unique=on&format=html&rnd=new).     |
 
 After setting these variables, you can deploy your application and Quirrel should be working.
 If it doesn't, feel free to [reach out](mailto:troubleshooting@quirrel.dev).
 
 :::note Cron Jobs
 If you're using [CronJobs()](/api/cronjob), make sure to run `quirrel ci` during the deploy process.
+Make sure to have `QUIRREL_API_URL`, a `QUIRREL_TOKEN` and the `QUIRREL_BASE_URL` set when executing `quirrel ci`.
 
 ```json
 "scripts": { "vercel-build": "npm run build && quirrel ci" }
 ```
 
 :::
 
-:::note VERCEL_URL
+:::note VERCEL*URL
```

Comment:
this is so confusing. in my ide:

![image](https://user-images.githubusercontent.com/4662748/196870269-b2ba4c21-bb50-4a0e-9d4b-5e92c9990fa1.png)